### PR TITLE
FISH-6957 Update Documentation for Payara 6 Enterprise

### PR DESCRIPTION
**Pull request by Pandrex247**
@fturizo you said you were going to do your own scan of things.
To save some redundancy, here's what I've changed so far.

### CODE REVIEW

**Comment by fturizo** [comment]
I have expanded on the initial set of changes that we discussed a few weeks ago and added them to your PR, @Pandrex247, could you and maybe @JamesHillyard take a look into them and let me know if they are OK or if something is correct or missing? 

Explicitly I haven't added any extra details to the **Notifiers** section as it seems there shall be no issues with the current batch of notifiers for 6's first release.

**Review by fturizo — APPROVED** [review]
Added extra details based on recent review of documentation for Payara Enterprise 6.x

File: enterprise/docs/modules/ROOT/pages/General Info/Supported Platforms.adoc [hunk 1]
```diff
@@ -1,42 +1,39 @@
...
 
-The Payara Platform currently supports the following Java Virtual Machines:
-
-* Azul Platform Core JDK: 8 (u162+), 11 (11.0.12+), 17 (17.34/17.0.3+)
-* Oracle JDK: 8 (u162+), 11 (11.0.12+), 17 (17.0.3+)
-* Amazon Corretto: 8, 11 (11.0.12+), 17 (17.0.3+)
-* Adopt Open JDK: 8, 11 (11.0.12+), 17 (17.0.3+)
-* Adopt Open JDK with Eclipse Open J9: 8, 11 (11.0.12+), 17 (17.0.3+)
-* RedHat JDK: 8 (u162+), 11 (11.0.12+), 17 (17.0.3+)
+The Payara Platform currently runs the following Java Virtual Machines:
```
Comment (Pandrex247) [inline enterprise/docs/modules/ROOT/pages/General Info/Supported Platforms.adoc:9]: Payara doesn't run the JVM 😜
```suggestion
The Payara Platform currently runs on the following Java Virtual Machines:
```

File: enterprise/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Upgrade Payara/Upgrade Tool.adoc [hunk 2]
```diff
@@ -2,9 +2,12 @@
 = Payara Server Upgrade Tool
 :ordinal: 1
 
-This tool can be used to upgrade an existing Payara Server installation to a newer version. The Upgrade Tool is bundled in Payara Server versions starting from release `5.24.0` and is maintained as a separate JAR artifact under the `fish.payara.extras: payara-upgrade-tool` GAV coordinates.
+IMPORTANT: The Upgrade tool doesn't support migrating from Payara Server Enterprise 5.x installations to 6.x. +
```
Comment (Pandrex247) [inline enterprise/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Upgrade Payara/Upgrade Tool.adoc:5]: It also doesn't currently allow migrating from a 6.x to a newer 6.x

**Comment by fturizo** [comment]
@Pandrex247, the PR is ready to be merged, please take it out of Draft mode.

**Pull request merged**